app/src/views/features/sessions/ShareRecordingModal/components/StartFromOffsetInput/StartFromOffsetInput.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from "react";
+import { Checkbox } from "antd";
+import { RQInput } from "lib/design-system/components";
+import "./startFromOffsetInput.css";
+
+interface StartFromOffsetInputProps {
+  currentOffset?: string;
+  onOffsetChange: (offset: string | null) => void;
+}
+
+export const StartFromOffsetInput: React.FC<StartFromOffsetInputProps> = ({
+  currentOffset = "0:00",
+  onOffsetChange,
+}) => {
+  const [isChecked, setIsChecked] = useState(false);
+  const [offset, setOffset] = useState(currentOffset);
+
+  const handleOffsetChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setOffset(e.target.value);
+    if (isChecked) {
+      onOffsetChange(e.target.value);
+    }
+  };
```

Comment:
Add a check for a valid duration before updating, currently we can add negative value or any value which is greater then the duration itself